Android/app/src/main/res/layout/activity_my_trip_info.xml

```diff
@@ -29,7 +29,7 @@
                 android:contentDescription="@string/empty_description"
                 android:scaleType="centerCrop"
                 android:src="@drawable/placeholder_image"
-                android:visibility="gone"
+                android:visibility="visible"
```

Comment:
why this change?

Reply:
It was changed for just testing. Changes in next new commits

Reply:
You havent changed it yet

Reply:
Done in new commit